promising/configs.py

```
@@ -0,0 +1,195 @@
+        parent_config: Optional["PromiseConfig"] | Sentinel = NOT_SET,
+        *,
+        start_soon: bool | Sentinel = NOT_SET,
+        make_parent_wait: bool | Sentinel = NOT_SET,
+        config_inheritable: bool | Sentinel = NOT_SET,
+    ) -> None:
+        self._parent_config = None
+        self._inheritable_parent_config = None
+
+        if parent_config is NOT_SET:
+            try:
+                # ruff: noqa: PLC0415 (import-outside-top-level)
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)